src/script/redux/reducers.js

```diff
@@ -0,0 +1,7 @@
+const { simpleObject } = require('trampss-redux-factory')
+
+module.exports = {
+  component: simpleObject('component'),
+  project: simpleObject('project'),
+  config: simpleObject('config'),
```

Comment:
I don't see the purpose in server side, since we are not in a reactive way of programming.